precompiles/src/precompile_set.rs

```diff
@@ -36,6 +35,10 @@ use impl_trait_for_tuples::impl_for_tuples;
 use pallet_evm::AddressMapping;
 use sp_core::{H160, H256};
 
+// This is the simplest bytecode to revert without returning any data.
+// (PUSH1 0x00 PUSH1 0x00 REVERT)
+pub const REVERT_BYTECODE: [u8; 5] = [0x60, 0x00, 0x60, 0x00, 0xfd];
```

Comment:
It seems that we don't need this dummy bytecode anymore, you removed its usage

Reply:
It is moonbeam specific, used only for the old foreign assets. Makes sense that we remove it from the frontier side